rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -35,6 +35,7 @@ type AssetIdentifier: <"The unique identifier for an Asset, specified using an A
     identifierType AssetIdTypeEnum (1..1) <"Defines the symbology source of the Asset Identifier, eg CUSIP, ISIN, etc.">
 
 type InstrumentBase extends AssetBase: <"Defines the common attributes for all Instrument data types.">
+    instrumentType InstrumentTypeEnum (1..1) <"Identifies the type of an instrument using an enumerated list.">
```

Comment:
Moving this mandatory attribute from `Security` up to `InstrumentBase` causes an increase in validation failures in the `InstrumentBase` sub types `Loan` and `ListedDerivative`